electionleaflets/apps/leaflets/filters.py

```diff
@@ -0,0 +1,38 @@
+import django_filters
+from dc_utils.filter_widgets import DSLinkWidget
+
+
+def region_choices():
+    """
+    Return a list of tuples with NUTS1 code and label. Used by the region filter
+    on the BaseBallotFilter
+    """
+    return [
+        ("UKC", "North East"),
```

Comment:
As an enhancement, it would be quite nice if this was abstracted out so that we could use it both as a `choices=` on the model (so if we edit a leaflet in the admin, we get a dropdown to pick from) and also pull it in on the filter here.

Reply:
Yeah, good idea. I've just pushed this up